packages/utils/package.json

```diff
@@ -36,15 +62,17 @@
     "test:browsers": "yarn karma start karma.config.cjs",
     "check-readme": "typescript-docs-verifier"
   },
-  "types": "lib/index.d.ts",
   "dependencies": {
     "@chainsafe/as-sha256": "^0.3.1",
     "any-signal": "2.1.2",
     "bigint-buffer": "^1.1.5",
     "case": "^1.6.3",
     "chalk": "^4.1.2",
     "js-yaml": "^3.13.1",
-    "winston": "^3.3.3"
+    "winston": "^3.3.3",
+    "prom-client": "^14.1.0",
```

Comment:
prom-client is heavy dependency assumed to be run in NodeJS, why is this dependency necessary? Please remove